admin/sqlpatch.php

```diff
@@ -833,7 +833,7 @@ function init() {
           <div class="col-sm-12 text-danger"><strong><?php echo HEADING_WARNING2; ?></strong></div>
         </div>
         <?php
-        if ($action == 'execute' && $_POST['query_string'] != '') {
+        if ($action == 'execute' && isset($_POST['query_string']) && $_POST['query_string'] != '') {

```

Comment:
Why not just use `!empty($_POST['query_string'])`?

Reply:
Didn't take the opportunity to investigate a `falsey` `'query_string'` that was appropriate for processing.

Reply:
I don't understand your response.

Reply:
Why not? Because I hadn't evaluated if a problem would occur by using `!empty`. Still haven't, but responded at least.

Reply:
In this case, if a falsey value was all that was posted, it would never be a valid query, so `empty()` would be fine.